### ISSUE TITLE
Software Features To Add

### ISSUE THREAD

**Issue by Madness44** [PancakeBot/PancakeCreator#55]
The software is simple, and i like that. However i feel there are some basic features that need to be added:

Undo/Redo - Others have also mentioned it, nothing worse than deleting an outline you just worked on

Eraser - I know you can select and delete, but a simple eraser to quickly fix those missed freehand outlines. Also allow different sizes/shapes

Simple shapes - add things like circles, stars, ect. that can be easily and evenly made without having to freehand. 

Text - add simple text  so once again it can be somewhat even and alot easier to create.

Basically look at Windows Paint. Add many of those features while still keeping it simple

**Comment by AstroJetson** [comment]
Let me add to Madness44 list
-- Text - Need to have the ability to outline text in one color and then fill with another 
-- Shapes -- Need to be able to outline in one color and then fill with another

**Comment by migpics** [comment]
I agree with these all.  Getting the list ready.
